### PR TITLE
cabana: add cajita variant

### PR DESCRIPTION
**Pull request by eugeneswalker**
`cabana` : add `cajita` variant (default to off)

@junghans @sslattery @streeve @wspear

### CODE REVIEW

File: var/spack/repos/builtin/packages/cabana/package.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Cabana(CMakePackage):
     variant('arborx', default=False, description='Build with ArborX support')
     variant('heffte', default=False, description='Build with heFFTe support')
     variant('hypre', default=False, description='Build with HYPRE support')
+    variant('cajita', default=False, description='Build with Cajita support')
```
Comment (streeve) [inline var/spack/repos/builtin/packages/cabana/package.py]:
```suggestion
    variant('cajita', default=False, description='Build Cajita subpackage')
```

File: var/spack/repos/builtin/packages/cabana/package.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ class Cabana(CMakePackage):
...
 
     depends_on("cmake@3.9:", type='build')
     depends_on("googletest", type='build')
```
Comment (streeve) [inline var/spack/repos/builtin/packages/cabana/package.py:41]: Also `depends_on` MPI when Cajita is enabled

**Review by junghans — DISMISSED** [review]

**Review by streeve — CHANGES_REQUESTED** [review]
Thanks for the update

File: var/spack/repos/builtin/packages/cabana/package.py (hunk 2 above)
Comment (eugeneswalker) [reply var/spack/repos/builtin/packages/cabana/package.py:41]: Got it! Change pushed: 638fe807c2c6b5bca2ffc88e7ec340f94066a316

**Review by streeve — APPROVED** [review]

**Review by wspear — APPROVED** [review]
Looks good.

**Pull request merged**